File: app-banco-frontend/src/app/pages/_dto/page-DTO.ts
/**
 * Interface GenericResponseDTO
 * @autor Juan Maigua <maiguarizocarlos@gmail.com>
 * @since 22/05/2026
 * @version 1.0
 */
export interface PageDTO<T> {
  page: number;
  size: number;
  tabla: T;
}

File: dependecy/person-client-account-movement-dependency/src/main/java/com/personclientaccountmovement/dto/PageDTO.java
package com.personclientaccountmovement.dto;

import java.io.Serializable;

import lombok.Data;

/**
* DTO Paginación de una entidad
* 
* @author Juan Maigua <mailto:maiguarizocarlos@gmail.com>
* @version 1.0 - Version Inicial
* @since 15/04/2026
*/
@Data
public class PageDTO<T> implements Serializable {
	private static final long serialVersionUID = 1L;
	private Integer page;
	private Integer size;
	private String order;
	private String orderValue;
	private T tabla;
	private String nombreFiltroCons = "";
	private String valorFiltroCons = "";
}
